Add SUMO Scheduler

maximbelyakov opened the pull request:
**Type of pull request:** product addition
**Related issues:** list the related issues here (if applicable)

maximbelyakov committed:
add sumo-scheduler

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,78 @@
+[rubric.behavioral-marketing]
+value = "yes-opt-out"
+citations = [
+  "Yes, but you may opt-out. If you no longer wish to receive any marketing-related emails from us, you may opt-out via the unsubscribe link included in such emails."
```
This doesn't seem to be a direct citation and would only cover opt-out from newsletters, not targeted advertising

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,78 @@
+[rubric.security]
+value = "yes"
+citations = [
+  "Yes. For detailed information on our data security practices, please refer to our Privacy Policy. https://sumoscheduler.com/privacy-policy/"
```
Not a citation. Can quote some of the things in the « How do we protect your information ? » section, but the score looks like it would end up being `somewhat` given it's the usual basic precautions most other policies mention

doamatto replied:
Now that I've seen the security policy a bit farther down, quoting from there and adding a note to check it would be ideal.

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,78 @@
+[rubric.third-party-collection]
+value = "yes"
+citations = [
+  "Yes, but only critical data (name, phone number, email). We collect and store content that you create, submit, post, upload, transmit, store or display in the process of using SUMO or our websites."
```
This is first, not third party, collection. Should get rescored and a quote to match

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,78 @@
+[rubric.history]
+value = "last-modified"
+citations = [
+  "Yes"
```
Citation should be of last modified in privacy policy (`These updates will take effect on May 25, 2018.`)

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,78 @@
+[rubric.data-deletion]
+value = "yes-contact"
+citations = [
+  "Yes, by contacting someone. If your account is managed by a System Administrator, that administrator may have control with regards to how your account information is retained and deleted. However, customers may also contact us at support@sumoscheduler.com to request the complete deletion of your personal data. We will respond to your request within 30 days."
```
Remove « Yes, by contacting someone. » since that's not a quote on the page

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,78 @@
+[rubric.third-party-access]
+value = "yes-specified-noncritical"
+citations = [
+  "No, we do not sell, trade, or otherwise transfer to outside parties your personally identifiable information."
```
Data seems to be shared for advertising, payment processing, and analytics from a quick glance. Score and citation should be updated to reflect — no subprocessors disclosed it seems either, bar Salesforce as the « data hosting provider ».

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,78 @@
+]
+
+[rubric.data-collection-reasoning]
+value = "yes"
```
I'd bid for `mostly` given the vague « For other purposes about which we obtain your consent. » part at the end

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,78 @@
+[rubric.data-collection-reasoning]
+value = "yes"
+citations = [
+  "We collect and store content that you create, submit, post, upload, transmit, store or display in the process of using SUMO or our websites."
```
Would be ideal to have the full list of collection purposes

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,78 @@
+[rubric.noncritical-purposes]
+value = "opt-out-some"
+citations = [
+  "Yes, but you may opt-out. If you no longer wish to receive any marketing-related emails from us, you may opt-out via the unsubscribe link included in such emails."
```
Not a direct citation. May be a good idea to add a note mentioning that you're only able to opt-out of e-mails

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,78 @@
+[rubric.law-enforcement]
+value = "strict"
+citations = [
+  "Only when required by a court order or subpoena. We may disclose your information, including your personal data, to a third party if (a) we believe that disclosure is reasonably necessary to comply with any applicable laws and regulation or governmental request by court order or subpoena."
```
First sentence isn't a citation and should be removed

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,78 @@
+[rubric.revision-notify]
+value = "yes"
+citations = [
+  "No. As per the policy, SUMO reserves the right to amend our Privacy Policy at any time to reflect changes in the law, our data collection and use practices, and the features of our Services. If we decide to change our privacy policy, we will post those changes on this page. https://sumoscheduler.com/privacy-policy/"
```
Seems the privacy policy mentions that an e-mail will be sent out — should be updated to reflect. Score is OK

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,78 @@
+[rubric.list-collected]
+value = "generally"
+citations = [
+  "Yes, but only critical data (name, phone number, email). We collect and store content that you create, submit, post, upload, transmit, store or display in the process of using SUMO or our websites."
```
Should be a citation(s) of the full list of data collected

doamatto reviewed (changes requested):
Here's a few things that need fixing, but a good start

maximbelyakov committed:
update sumo-scheduler on Code Review comments

maximbelyakov committed:
update sumo-scheduler on Code Review comments #2

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,88 @@
+
+[rubric.security]
+value = "yes"
+citations = [
```
Can't seem to find these citations. Can move the security policy mention to a note.

maximbelyakov replied:
https://sumoscheduler.com/privacy-policy/

I used "How do we protect your information?", the paragraphs: This section discusses the implementation of security measures such as SSL encryption and secure server protocols to safeguard personal information.

Please check

doamatto replied:
This is summarising that section, rather than citing it. Summary is fine, but it should be in the notes instead.

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,88 @@
+[rubric.third-party-collection]
+value = "yes"
+citations = [
+  "We may receive Personal Information about you from other sources, including through third-party services and organizations to supplement information provided by you."
```
Citation not in privacy policy

maximbelyakov replied:
Section: "What information do we collect?"
Paragraphs: This section mentions that SUMO may collect information from third-party services to supplement user-provided information.

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,88 @@
+[rubric.data-deletion]
+value = "yes-contact"
+citations = [
+  "If your account is managed by a System Administrator, that administrator may have control with regards to how your account information is retained and deleted. However, customers may also contact us at support@sumoscheduler.com to request the complete deletion of your personal data. We will respond to your request within 30 days."
```
Citation not in privacy policy

maximbelyakov replied:
Section: "YOUR CHOICES" and "Personal Data Access"
Paragraphs: These sections describe how users can contact support to request the deletion of their personal data and mention the response time for such requests.

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,88 @@
+[rubric.third-party-access]
+value = "yes-specified-noncritical"
+citations = [
+  "We may share your information with third parties that perform services for us or on our behalf, including payment processing, data analysis, email delivery, hosting services, customer service, and marketing assistance.",
```
Can't seem to find citation

maximbelyakov replied:
Setion: "Do we disclose any information to outside parties?"
Paragraphs: This section explains that information may be shared with third parties for specific services like payment processing and analytics, provided they maintain confidentiality.

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,88 @@
+
+[rubric.data-collection-reasoning]
+value = "mostly"
+citations = [
```
Can't find citations

maximbelyakov replied:
Section: "What do we use your information for?"
Paragraphs: Thhis section outlines the purposes for which data is collected, including improving services, processing transactions, and legal compliance.

doamatto replied:
This is summarising that section, rather than citing it. Summary is fine, but it should be in the notes instead.

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,88 @@
+[rubric.list-collected]
+value = "generally"
+citations = [
+  "The personal information that we collect depends on the context of your interactions with us and the Services, the choices you make and the products and features you use. The personal information we collect can include the following: Name and Contact Data, Credentials, Payment Data, Device Data, Location Data, Usage Data.",
```
Can't find citation

maximbelyakov replied:
Section: "What information do we collect?"
Paragraphs: This section lists the types of personal information collected, such as name, contact details, payment information, and usage data.

doamatto commented on products/sumo-scheduler.toml:
```diff
@@ -0,0 +1,88 @@
+[rubric.revision-notify]
+value = "yes"
+citations = [
+  "If we make material changes to this privacy policy, we will notify you by email or by means of a notice on our homepage prior to the change becoming effective."
```
Can't find citation

maximbelyakov replied:
Section: "Changes to our Privacy Policy"
Paragraphs: This section describes how users will be notified of material changes to the privacy policy, such as through email or a notice on the homepage.

doamatto committed:
Merge branch 'master' into master

maximbelyakov commented:
@doamatto Could you check please?

doamatto reviewed (dismissed):
I assume from your GH profile you work here and the quotes might be from an upcoming change. Once it's reflected there, I'll give it another once over to make sure everything's solid in that regard. Until then, I can't merge it since most of the citations don't exist in the policy.

maximbelyakov committed:
update sumo-scheduler on Code Review comments by Nov 25

maximbelyakov commented:
@doamatto Hello!

Could you review the changes please?

maximbelyakov commented:
@doamatto Do you need any changes from me?

maximbelyakov commented:
@bcbee @loviuz @milesmcc 

Sorry, can you check it please guys?

doamatto reviewed (commented):
Apologies for the delays — I had more pressing matter to attend to recently. This should be everything, but I'll polish it and get this MR merged in a titbit. Thank you for your work

doamatto committed:
fix(product): changes to SUMO Scheduler

- Add citation and update score for rubric.security
- Remove missing citation, add note, update score for rubric.third-party-collection
- Add real citation for rubric.data-deletion
- Move link for rubric.data-breaches
- Update score for and cite rubric.third-party-access
- Add real citation for rubric.data-collection-reasoning
- Remove superfluous note for rubric.noncritical-purposes
- Update score for rubric.law-enforcement
- Add real citation for rubric.list-collected
- Add real citation for rubric.revision-notify
